developer_manual/design/foundations.rst

```diff
@@ -4,9 +4,13 @@ Foundations
 
 There are several design elements that are common to all Nextcloud apps. If you are developing for a platform that has its own design specifications, for example Android, it would be a good idea to keep those in mind while designing your app.
 
+For web apps, there is the `Vue library <https://nextcloud-vue-components.netlify.app/>`_ and the `Penpot Design Kit <https://design.penpot.app/#/view/db3839da-807b-8052-8002-576401e9a375?page-id=db3839da-807b-8052-8002-576401e9a376&section=interactions&index=0&share-id=88d87192-d205-8007-8002-8ebfad297c28>`_.
```

Comment:
Just FYI from this link it was only possible to view the cover page. I adjusted the link settings so that everything is viewable (only the "Nextcloud design kit (current)" was checked before):
![image](https://github.com/nextcloud/documentation/assets/925062/a0b85c86-1b00-45f4-bc2f-906a6ef4a8b6)

Reply:
I fear there is a bug. I just tried my link above in a new tab and still only the cover is shown. Also the additional share permissions are lost after a reload IIRC?

Reply:
Link change so it’s a consistent link with the others as well:
```suggestion
For web apps, there is the `Vue library <https://nextcloud-vue-components.netlify.app/>`_ and the `Penpot design kit <https://design.penpot.app/#/view/db3839da-807b-8052-8002-576401e9a375?page-id=db3839da-807b-8052-8002-576401e9a376&section=interactions&index=0&share-id=11fde340-21f4-802e-8002-8d8d305e7ab5>`_.
```

Reply:
done